### DISCUSSION TITLE
S3 (Credentials from AWS Security Token Service) connection profile not available

### DISCUSSION THREAD

**Discussion by cpalermo-sharper** [iterate-ch/cyberduck#15847] [Q&A]
Hi all,

I'm setting up a Cyberduck instance to connect to our S3 cloud, but I'm facing an issue. I used to be able to connect to it using the S3 (Credentials from AWS Security Token Service) connection profile, but for some reason it isn't available any longer on the cyberduck server:
[S3 (Credentials from AWS Security Token Service) ](https://svn.cyberduck.io/trunk/profiles/S3%20(Credentials%20from%20AWS%20Security%20Token%20Service).cyberduckprofile)
Yet it's stioll available as a link on the website.
![Screenshot 2024-04-18 154503](https://github.com/iterate-ch/cyberduck/assets/142983102/0d6bad27-07b3-4589-88be-eda60534a7b4)

Can anyone help me find this connection profile?

**Comment by dkocher** [comment]
It was removed in [^1] as it is identical with the _S3 (Credentials from AWS Command Line Interface)_ connection profile.

[^1]: https://github.com/iterate-ch/profiles/commit/2a577851e8aed0097cb3d362ba95a32f3fa5f850
